low-level/files/opt/seagate/sspl/setup/sspl_config.py

```diff
@@ -0,0 +1,327 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+#################################################################
+# This script performs following operations
+# - Check if product is one of the enabled products
+# - Configures role in sspl.conf if supplied
+# - Executes sspl_reinit script
+# - Updates cluster nodes passed in cli to consul
+#################################################################
+
+import os
+import sys
+import subprocess
+import shutil
+import psutil
+import json
+sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level/')
+
+import rpm
```

Comment:
use 'rpm' if available in direct python3 site packages only. or use SimpleProcess from cortxutils to manage rpms.